extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -473,6 +454,25 @@ export class OpenPGPKey {
     return secondTry ? secondTry : null; // tslint:disable-line:no-null-keyword
   }
 
+  private static arePrivateParamsMissing = (packet: OpenPGP.packet.BaseKeyPacket): boolean => {
```

Comment:
Are you opening the vscode project using the project file in the repo, or simply as a folder?

If you opened it as a project, it should prompt you to install needed extensions. One of them should do auto-formating on save, which would have moved the method for you automatically.

Reply:
Workspace file? I'll try that.

Reply:
Yup. The IDE should be a whole bunch smarter if you do that.